lib/src/uri_template.dart

```diff
@@ -428,7 +428,7 @@ class _Compiler {
           queryVariables.addAll(_parseMap(subpart, '&'));
         } else if ((subpart as Match).group(1) == '?') {
           throw ParseException('multiple queries');
-        } else if ((subpart as Match).group(1) == '#') {
+        } else if ((subpart).group(1) == '#') {
```

Comment:
nit: unnecessary parens now

Reply:
I have a follow-up cleanup commit for this!